lnwire/lnwire_test.go

```diff
@@ -864,6 +864,22 @@ func TestLightningWireProtocol(t *testing.T) {
 					NewShortChanIDFromInt(uint64(r.Int63())))
 			}
 
+			v[0] = reflect.ValueOf(req)
+		},
+		MsgPing: func(v []reflect.Value, r *rand.Rand) {
+			// We use a special message generator here to ensure we
+			// don't generate ping messages that are too large,
+			// which'll cause the test to fail.
+			//
+			// We'll allow the test to generate padding bytes up to
+			// the max message limit, factoring int he 2 bytes for
+			// the num pong bytes.
+			paddingBytes := make([]byte, r.Intn(MaxMsgBody-2))
+			req := Ping{
+				NumPongBytes: uint16(r.Intn(MaxPongBytes)),
```

Comment:
nit: same here, could be [0, MaxPongBytes] for more complete coverage of possible values